LayoutControl.tis

```diff
@@ -85,15 +81,14 @@ class LayoutControl
     $(#colorsContainer).remove();
   }
 
-  function removeBlock(colorBlock)
-  {
+  function removeBlock(colorBlock){
     colorBlock.@#toDelete = true; 
 		var container = $(#colorsContainer);
 		var i = 0;
 		for (i = 0; i < container.length; i++) {
 			if (container[i].@#toDelete == true) break;
 		}
-		LayoutControl.blockColorsList.remove(i-1);
+		LayoutController.blockColorsList.remove(i-1);
```

Comment:
you can use Array.removeByValue, so remove all block iterating code above, and just remove the provided colorBlock as is